src/openrct2-ui/windows/Staff.cpp

```diff
@@ -828,7 +828,12 @@ static Widget _staffOptionsWidgets[] = {
                     widgets[WIDX_CHECKBOX_4].type = WindowWidgetType::Empty;
                     widgets[WIDX_COSTUME_BOX].type = WindowWidgetType::DropdownMenu;
                     widgets[WIDX_COSTUME_BTN].type = WindowWidgetType::Button;
-                    widgets[WIDX_COSTUME_BOX].text = StaffCostumeNames[EnumValue(staff->SpriteType) - 4];
+
+                    // TODO: retrieve string from object instead
+                    if (EnumValue(staff->SpriteType) >= 4)
+                        widgets[WIDX_COSTUME_BOX].text = StaffCostumeNames[EnumValue(staff->SpriteType) - 4];
```

Comment:
NB: the problem is that peep/handyman/mechanic/security 'entertainers' are all defined before the panda costume (4), therefore indexing negatively on `StaffCostumeNames`. Yikes.